controlplane/internal/controller/clusterdeployment_controller.go

```diff
@@ -327,5 +328,14 @@ func (r *ClusterDeploymentReconciler) computeAgentClusterInstall(
 			ManifestsConfigMapRefs: additionalManifests,
 		},
 	}
+
+	if len(acp.Spec.AgentConfigSpec.APIVIPs) > 0 && len(acp.Spec.AgentConfigSpec.IngressVIPs) > 0 {
+		aci.Spec.APIVIPs = acp.Spec.AgentConfigSpec.APIVIPs
+		aci.Spec.IngressVIPs = acp.Spec.AgentConfigSpec.IngressVIPs
+		aci.Spec.PlatformType = hiveext.PlatformType(configv1.BareMetalPlatformType)
```

Comment:
Should we also expose `PlatformType` field in our AgentControlPlane spec?

Reply:
I feel like this detail is not really necessary. Do you have any usecase in mind where this would help?

Reply:
Just wondering in the event the user might want platformType none since that's still a viable option right? Unless we're really only limiting the scope to Sylva and they only want platformType baremetal

Reply:
Yes, platform none is still available. It will be used if we do not specify VIPs (and for SNO)

Reply:
Gotcha! Thanks for confirming!